sheets/utils.py

```diff
@@ -661,3 +662,28 @@ def build_multi_cell_update_request_body(row_index, column_index, values):
             "fields": "*",
         }
     }
+
+
+def build_drive_file_email_share_request(file_id, email_to_share):
+    """
+    Builds the body of a Drive file share request
+
+    Args:
+        file_id (str): The file id of the Drive file being shared
+        email_to_share (str): The email of the user to whom the file will be shared
+
+    Returns:
+        dict: A dictionary of parameters for the body of a share request
+    """
+    added_kwargs = (
+        {"sendNotificationEmail": False}
+        if GOOGLE_SERVICE_ACCOUNT_EMAIL_DOMAIN in email_to_share
```

Comment:
Can we use `.endswith` to be explicit here and in `get_credentials` in sheets/api? I think it will also help prevent against unintended matches if a user names their email something like `fake.iam.gserviceaccount.com.fake@gmail.com`